languages/elixir/exercises/concept/exceptions/.meta/design.md

```diff
@@ -0,0 +1,24 @@
+## Learning objectives
+
+- Know what the _Exception behaviour_ is
+- Know how to define your own exceptions
+  - with just :message
+  - with :message and exception/1
+- Know of `try..rescue`'s additional options:
+  - `after`
+  - `else`
```

Comment:
It would be pretty hard to design an exercise where it makes a difference if certain logic is in `else` or the `try` block itself, so we will need an analyzer rule to check that `else` was used.